ocaml/xapi/agility.ml

```diff
@@ -60,6 +60,10 @@ let caching_vm_t_assert_agile ~__context (ok_srs, ok_networks) vm vm_t =
   if vm_t.API.vM_VGPUs <> [] then
     raise (Api_errors.Server_error
              (Api_errors.vm_has_vgpu, [Ref.string_of vm]));
+  (* Any kind of VUSB means that the VM is not agile. *)
+  if vm_t.API.vM_VUSBs <> [] then
+    raise (Api_errors.Server_error
+             (Api_errors.vm_has_vusbs, [Ref.string_of vm]));
```

Comment:
This is true now, but we'll have to revisit this when we drop the restrictions on USB_groups only having 1 PUSB and 1 VUSB